oprf/oprf.go

```diff
@@ -195,8 +193,18 @@ func (s *Server) Evaluate(b BlindToken) (*Evaluation, error) {
 	return &Evaluation{ser}, nil
 }
 
+func mustWrite(h io.Writer, data []byte) {
+	dataLen, err := h.Write(data)
+	if err != nil {
```

Comment:
Write from Hash will never error, why is this checked then?

https://golang.org/pkg/hash/#Hash

```
    // Write (via the embedded io.Writer interface) adds more data to the running hash.
    // It never returns an error.
    io.Writer
```

Reply:
Defensive programing, in case it does. :)